packages/mask/src/plugins/Savings/SNSAdaptor/SavingsDialog.tsx

```diff
@@ -97,12 +103,46 @@ export function SavingsDialog({ open, onClose }: SavingsDialogProps) {
         chainId,
     )
 
+    const { value: yfiTokens } = useAsync(async () => {
+        if (!isValidYearnChain(chainId)) {
+            return []
+        }
+
+        // @ts-ignore
```

Comment:
why?

Reply:
because Yearn does not accept all chains

Reply:
plz add reason to the comment, i don't accept no reason use `@ts-ignore`

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-6.html#suppress-errors-in-ts-files-using--ts-ignore-comments

Reply:
Mask supports more chains than Yearn. Passing those chains to yearn will throw errors